specification/databricks/resource-manager/Microsoft.Databricks/preview/2024-09-01-preview/accessconnector.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "DatabricksAccessConnectorClient",
-    "version": "2024-05-01",
+    "version": "2024-09-01-preview",
```

Comment:
@curiekim00 Can you check with Subbu or Yuqing as to whether we need to update Access Connector?

Reply:
Hi @scottmacchia-msft checked with Yuqing, there is no pending changes in the Access Connector, so I have reverted the Access Connector version update and pointed to the 2024-05-01 version, will double check with Subbu again when he is back

Reply:
You should move all your APIs forward together per uniform versioning. This is not an absolute requirement yet, but some SDKs can't be generated with mixed API versions, so uniform versioning is highly recommended.

Moving everything forward with each new API version makes consuming your APIs and SDKs much easier for customers.

Reply:
Hi @mentat9 I have upgraded the API version for Access Connector as well, but it looks like now the bot has added the Breaking Change tag despite it not being breaking change before the removal.

Reply:
Yes, the Python SDK team will help you with that. The fix guide link in the Next Steps to Merge comment tells you how to proceed.